### DISCUSSION TITLE
mspider/user-guide/gateway-instance/delete/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#3363] [Announcements]
# mspider/user-guide/gateway-instance/delete/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/gateway-instance/delete/

**Comment by nicole-lihui** [comment]
删除网关之前，需要检查相应的 Gateway 和 VirttualService 资源，否则会有无效配置，导致流量异常

**Comment by windsonsea** [comment]
Fixed in #3504
